src/hooks/useBestTrade.ts

```diff
@@ -34,7 +38,7 @@ export function useBestTrade(
 
   const [debouncedAmount, debouncedOtherCurrency] = useDebounce(
     useMemo(() => [amountSpecified, otherCurrency], [amountSpecified, otherCurrency]),
-    DEBOUNCE_TIME
+    useDebounceSwapQuoteFlag() === DebounceSwapQuoteVariant.Enabled ? DEBOUNCE_TIME_INCREASED : DEBOUNCE_TIME
```

Comment:
i wish there was a way to control the actual integer value here from statsig, but from what i can tell it's boolean only.

Reply:
we could do it via an experiment but i didnt really wanna mess with that atm